packages/modules/manage-versions/src/components/ManageVersions/ManageVersions.tsx

```diff
@@ -235,6 +320,7 @@ export const ManageVersions = (props: ManageVersionsProps) => {
               onVersionUpdated={refreshVersions}
               loadMoreVersions={getMoreVersions}
               onViewClick={onViewClick}
+              isLoadingColumn={!usersRef.current}
```

Comment:
does it even work?
Ref does not trigger a rerender...

Reply:
removed this prop , on update state with user name will already trigger render so just checking user name should not be empty.
Assuming creator info must be associated with the data, if not then there will be some issue with backend and will show skeleton